index.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+/* eslint-disable import/no-cycle */
+import addToList from './modules/add.js';
+import Books from './modules/books.js';
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.date = Book.timeStamp();
+    this.id = Math.random();
+  }
+
+  static timeStamp() {
+    const date = new Date();
+    return date.getMinutes();
+  }
+}
+
+const books = new Books();
+
+function getInput() {
+  const title = document.getElementById('title');
+  const author = document.getElementById('author');
+  const book = new Book(title.value, author.value);
+  title.value = '';
+  author.value = '';
+  return book;
+}
```

Comment:
It will be better to make this function an `es6 syntax` function. Remember we need to use `es6` for this milestone. 

**HINT**
```
const timeStamp = () => { your code }
```